packages/pass-style/src/types.js

```diff
@@ -42,10 +57,26 @@ export {};
  * using 'slots').
  */
 
+// Cases match in sequence. The final case 'remotable' is for a Passable that isn't one of the others.
 /**
- * @callback PassStyleOf
- * @param {Passable} passable
- * @returns {PassStyle}
+ * @typedef {{
+ * (p: undefined): 'undefined';
+ * (p: string): 'string';
+ * (p: boolean): 'boolean';
+ * (p: number): 'number';
+ * (p: bigint): 'bigint';
+ * (p: symbol): 'symbol';
+ * (p: null): 'null';
+ * (p: Promise): 'promise';
+ * (p: Error): 'error';
+ * (p: CopyTagged): 'tagged';
+ * (p: Array): 'copyArray';
+ * (p: Iterable): 'remotable';
+ * (p: Iterator): 'remotable';
```

Comment:
I don't understand why `Iterable` and `Iterator` are here. I wonder what I'm missing.

Reply:
hacks along the way. I'll see about removing them

Reply:
I didn't know how else to handle these cases https://github.com/endojs/endo/blob/3d927c30dec4786d16aa1e988e86d0ea6617f2e2/packages/patterns/test/test-copyMap.js#L131-L133

Reply:
oh. wild. I didn't realize `getCopyMapEntries()` returned a passable value.

I _did_ know that `tsc` thinks `getCopyMapEntries()` returns `any` before this PR.
https://github.com/Agoric/dapp-offer-up/blob/8da2191af4a3333e0ec0f0cd884f739fa021950a/contract/src/gameAssetContract.js#L21